src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -1126,6 +1131,12 @@ export class ResultsViewPageStore {
                     }
                 );
             }
+            // add counts for custom chart clinical attributes
+            for (const attr of this.clinicalAttributes_customCharts.result!) {
+                ret[attr.clinicalAttributeId] = attr.data!.filter(d => {
+                    return sampleMap.has(d as any, ['sampleId', 'studyId']);
```

Comment:
Looks like samples are already filtered in `makeCustomChartData`, do you still need this?